### PR TITLE
image

### PR DESCRIPTION
**Pull request by arundhatij12**

### CODE REVIEW

File: image.css [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+  flex-wrap: wrap;
+  height: max-content;
+  margin-top: 50px;
+}
+
+.content1 {
+  margin-left: 115px;
+  align-items: center;
+  justify-content: center;
+}
```
Comment (srejitk) [inline image.css:49]: This class seems to be repeated in quite a few of your files, try to seperate the common properties

File: image.css [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+  padding: 10px;
+  position: relative;
+  top: 20px;
+}
+
+.imgsec {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
```
Comment (srejitk) [inline image.css:95]: Suggesting to make use of utility classes here

File: image.css [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+.img {
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+#card1 {
+  background-color: rgb(159, 189, 184);
+  height: 320px;
+  width: 300px;
```
Comment (srejitk) [inline image.css:108]: Make use of relative units and save color codes as Variables.

**Review by srejitk — APPROVED** [review]
Suggested a few changes, try to use more of utility classes, Relative Units, Appropriate Class Names and Variables for Color codes.

File: image.css [hunk 4]
```diff
@@ -0,0 +1,208 @@
+body {
+  background: linear-gradient(to bottom, #66ffcc 0%, #66ccff 105%);
+  font-family: "Gill Sans", "Gill Sans MT", Calibri, "Trebuchet MS", sans-serif;
+}
+
+header {
+  background-color: rgba(114, 195, 202, 0.575);
```
Comment (rahulcs754) [inline image.css:7]: try to color variable

File: image.css [hunk 5]
```diff
@@ -0,0 +1,208 @@
...
+  height: 80px;
+}
+
+a {
+  text-decoration: none;
+  color: #3d6dc5;
```
Comment (rahulcs754) [inline image.css:13]: use color variable

File: image.css [hunk 6]
```diff
@@ -0,0 +1,208 @@
...
+
+@media screen and (max-width: 1129px) {
+  .content1 {
+    display: none;
+  }
+}
+
+.head {
+  display: flex;
+  padding: 32px 6px;
```
Comment (rahulcs754) [inline image.css:28]: use rem unit for responsive

**Review by rahulcs754 — APPROVED** [review]
Please try to complete color variable

**Pull request merged**